src/libs/utils.js

```diff
@@ -57,6 +57,29 @@ export const connectAtom = (theAtom, name) => WrappedComponent => {
   return Wrapper
 }
 
+/**
+ * Sets up the given atom to sync to/from sessionStorage at the given key.
+ * On initialization, if the key exists, the value will be read in.
+ * If it doesn't, the current value of the atom will be written out.
+ */
+export const syncAtomToSessionStorage = (theAtom, key) => {
+  theAtom.subscribe(v => {
+    if (v === undefined) {
+      sessionStorage.removeItem(key)
+    } else {
+      sessionStorage[key] = JSON.stringify(v)
+    }
+  })
+  const existing = (() => {
```

Comment:
Why not use `_.has`?
```js
const existing = _.has(key, sessionStorage) ? JSON.parse(sessionStorage[key]) : undefined
```

Reply:
The try/catch also handles JSON parse errors, in case some bad data gets poked in.